src/test/java/com/microsoft/azure/spring/data/documentdb/repository/DocumentDBAnnotationIT.java

```diff
@@ -80,5 +89,27 @@ public void testDocumentDBAnnotationIT() {
         TestUtils.testIndexingPolicyPathsEquals(policy.getIncludedPaths(), TestConstants.INCLUDEDPATHS);
         TestUtils.testIndexingPolicyPathsEquals(policy.getExcludedPaths(), TestConstants.EXCLUDEDPATHS);
     }
+
+    @Test
+    @SneakyThrows
```

Comment:
Why we use `SneakyThrows` here?

Reply:
TimeUnit will throw exception, and we'd like not to handle that here.

Reply:
Why not explicitly use `throws` in the method signature?

Reply:
I think that may looks simple and clean for test code.